src/content/energy-consumption/index.md

```diff
@@ -4,65 +4,66 @@ description: The basic information you need to understand Ethereum's energy cons
+| PoS Ethereum        |               0.0026                |             1x             | [source](https://carbon-ratings.com/eth-report-2022)                                                                                              |
 
-[Digiconomist provides whole-network energy consumption and carbon footprints for Bitcoin and Ethereum](https://digiconomist.net/ethereum-energy-consumption). At the time of writing this article, Bitcoin expends about 200 TWh/yr of energy and emits about 100 MT (megatons) of carbon per year, while generating about 32,000 T of electrical waste from obsolete hardware annually. In comparison, the total energy expenditure for securing Ethereum is closer to **0.01 TWh/yr**.
+Estimates of YouTube's energy expenditure have also been broken down by channel and individual videos. [Those estimates](https://thefactsource.com/how-much-electricity-does-youtube-use/) imply that people consumed >175 times more energy watching Gangnam Style in 2019 alone than the entire Ethereum network uses per year.
 
-<EnergyConsumptionChart />
+It is often very difficult to obtain accurate estimates for energy consumption, especially when an entity has a complex supply chain or deployment details that influence their efficiency. For example, we have included an upper and lower estimate for gold mining that differ by about 90 TWh. Estimates of energy consumption by Netflix range dramatically depending on the source. Their own self-reported estimates are about 20x smaller than an independent estimate - there is some discussion about the reasons for this on [Carbon Brief](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix). Similarly, Youtube has been estimated to expend about [244 TWh/yr](https://thefactsource.com/how-much-electricity-does-youtube-use/), although the energy consumption depends a lot on the type of device videos are streamed on and the energy-efficiency of underlying infrastructure such as data centers - suitable values for these parameters are hard to estimate so there is substantial uncertainty.
 
-The figure above shows the estimated annual energy consumption in TWh/yr for various industries (retrieved in June 2022).
-_Note that the estimates presented in the plot are from publicly available sources that have been linked to in the text below. They are
-illustrative and do not represent an official estimate, promise or forecast._
+The chart above also includes comparisons to Bitcoin and to "PoW Ethereum". Estimates for Bitcoin's energy consumption vary widely between sources and it is a topic that attracts a lot of nuanced [debate](https://www.coindesk.com/business/2020/05/19/the-last-word-on-bitcoins-energy-consumption/) about not only the amount of energy consumed but the sources of that energy and the related ethics.
```

Comment:
Just thinking, do we need to differentiate "Ethereum when it was under PoW" and "EthereumPoW (ETHW)"? I'm fine with not, just noting

Reply:
Oh is that still a thing @wackerow 😆?

Reply:
```suggestion
The chart above also includes comparisons to Bitcoin and to Ethereum when it used proof-of-work.  Estimates for Bitcoin's energy consumption vary widely between sources and it is a topic that attracts a lot of nuanced [debate](https://www.coindesk.com/business/2020/05/19/the-last-word-on-bitcoins-energy-consumption/) about not only the amount of energy consumed but the sources of that energy and the related ethics.
```